### DISCUSSION TITLE
Use Execa 5.1.1 if you're trying to set up the test runner

### DISCUSSION THREAD

**Discussion by Obapelumi** [adonisjs/core#3366] [Cookbooks]
## Why?
The latest version of execa (6.0.0) introduced some breaking changes to function names and also causes an ES Module import error detailed below. So if you're trying to use it with japa for testing install version 5.1.1 as it is more stable.

## Execa ES Module Import Error:
Error [ERR_REQUIRE_ESM]: require() of ES Module /[PATH//node_modules/execa/index.js from /[PATH]/japaFile.ts not supported.
Instead change the require of index.js in /[PATH]/japaFile.ts to a dynamic import() which is available in all CommonJS modules.

**Comment by Thiagoow** [comment]
Much thanks mate! Saved me a lot

**Reply by Obapelumi** [reply]
You're welcome
Happy to be of help 😊
